modules/olm-updating-index-image.adoc

```diff
@@ -34,21 +34,42 @@ endif::[]
 [source,terminal]
 ----
 $ opm index add \
-    --bundles <registry>/<namespace>/<new_bundle_image>@sha256:<digest> \//<1>
-    --from-index <registry>/<namespace>/<existing_index_image>:<tag> \//<2>
-    --tag <registry>/<namespace>/<existing_index_image>:<tag> \//<3>
-    --pull-tool podman <4>
+    --bundles <bundle_registry>/<bundle_namespace>/<new_bundle_image>@sha256:<digest> \
+    --from-index <mirror_registry>/<mirror_namespace>/<existing_index_image>:<existing_tag> \
```

Comment:
I'm not sure if `bundle_registry` and `mirror_registry` is necessary. One could argue that this template is confusing to those who are not on a disconnected env. I think just `registry` sufficiently communicates that if you are on a disconnected env, you should use your "mirror_registry". Same comment for the namespaces template.

Reply:
Heard. I made the changes and adjusted the description below.